app/src/main/java/com/babylon/wallet/android/presentation/transfer/assets/ChooseAssetsSheet.kt

```diff
@@ -94,8 +94,12 @@ fun ChooseAssetsSheet(
         },
         containerColor = RadixTheme.colors.gray5
     ) { padding ->
-        val selectedAssets = remember(state.targetAccount.spendingAssets) {
-            state.targetAccount.spendingAssets.map { it.address }
+        val selectedResources = remember(state.targetAccount.spendingAssets) {
+            state.targetAccount.spendingAssets.filterIsInstance<SpendingAsset.Fungible>().map { it.resourceAddress }
+        }
+
+        val selectedNFTs = remember(state.targetAccount.spendingAssets) {
+            state.targetAccount.spendingAssets.filterIsInstance<SpendingAsset.NFT>().map { it.item.globalId }
```

Comment:
since you separated maybe rename the `selectedResources` to `selectedFungibles`
or keep it as it was before `selectedAssets`